source/nodejs/adaptivecards/src/shared.ts

```diff
@@ -34,6 +34,7 @@ export class GlobalSettings {
     static enableFallback: boolean = true;
     static useWebkitLineClamp: boolean = true;
     static allowMoreThanMaxActionsInOverflowMenu: boolean = false;
+    static addPaddingToContainerWithBackgroundImage: boolean = true;
```

Comment:
@paulcam206 who is the owner to approve this kind of API review?
How about the name `extendBackgroundImageToContainerPaddingArea`

Reply:
@jwoo-msft I would assume @dclaux is the right person?:)

Reply:
It depends what the feature actually is. `addPaddingToContainersWithBackgroundImage` pretty explicitly means that padding is added to containers that have a background image, which is how things work today. So it seems like the right name for a flag that allows to not add padding to containers that have a background image.

`extendBackgroundToContainerPaddingArea` seems to mean that, when set to `false`, background images will simply not cover the padding of their container, but not that the container isn't going to get padding.

Reply:
Why is this feature needed in the first place? Is there a screenshot of the desired experience that cannot be accomplished without this?

This flag basically makes an AC authored for a host that has it set to false not render properly in a host that has it set to true (and the other way around). That isn't a good interop story, it doesn't seem like a good solution at first glance.

Reply:
@dclaux; this is needed because we have a customer who needs to have custom styling of button. the customer wants to have a button with green background and corner radius. they are trying to achieve this by setting background image in the column. with padding added, their button becomes bigger and the corner radius disappears. this change is designed to support only one specific host at the moment.

Reply:
@jwoo-msft Wow, that is just not the right way to enable the customer. It's a major hack.

There are likely multiple right ways to support this. Here are a couple of them:
- Enhance `Container`
  - Allow custom paddings in containers. This feature is already supported in JS, it just requires adding a `padding` property on `Container`. You can disable the capability by default via a global setting and only allow specific hosts to support it.
  - Plus adding a `borderRadius` property to `Container`, which is somethingv that we've discussed multiple times in the past
- Introduce a dedicated `Button` element

_Please_ reconsider this.